src/components/Blog/Blog.jsx

```diff
@@ -0,0 +1,79 @@
+import './Blog.sass';
+
+import React, {PureComponent, Fragment} from 'react';
+import {
+    Badge,
+    Card,
+    CardFooter,
+    CardBody,
+    CardTitle,
+    CardText,
+    CardImg,
+    Button,
+    Pagination,
+    PaginationItem,
+    PaginationLink
+} from 'reactstrap';
+
+import Author from "containers/AuthorContainer";
+
+export default class Blog extends PureComponent {
```

Comment:
loadPage = (pageNumber) => (e) => {
  const { loadPage } = this.props;
  loadPage(pageNumber);
}